src/lib/stellar/horizonQueries.js

```diff
@@ -38,7 +38,16 @@ export async function fetchAccount(publicKey) {
         } catch (err) {
             // @ts-ignore
             if (err.response?.status === 404) {
-                throw error(404, 'account not funded on network')
+                try {
+                    await fundWithFriendbot(publicKey)
+
+                    let account = await server.accounts().accountId(publicKey).call()
+                    return account
+                } catch (err) {
+                    throw error(500, {
+                        message: `Unable to fund account ${publicKey}: ${error.message}`,
```

Comment:
```suggestion
                        message: `Unable to fund account ${publicKey}: ${err.message}`,
```

this'll need to change to `err`, as well